readthedocs.yml

```diff
@@ -0,0 +1,19 @@
+# .readthedocs.yml
```

Comment:
The file name should start with a period I think.

Reply:
Yup, thank you!